Content.Server/Chemistry/Components/ChemMasterComponent.cs

```diff
@@ -37,6 +38,9 @@ namespace Content.Server.Chemistry.Components
     [ComponentReference(typeof(SharedChemMasterComponent))]
     public class ChemMasterComponent : SharedChemMasterComponent, IActivate
     {
+        [ViewVariables]
+        private FixedPoint2 _pillType = FixedPoint2.New(1);
```

Comment:
FixedPoint2 is for things like reagents & damage. It's basically just a float with 2 decimal points. There's no need to use it for the pill type, just use (u)int.